xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/table/livetabledate.js

```diff
@@ -46,8 +46,8 @@
         var input = $(element);
         var hidden = input.prev('input[type="hidden"]');
         var dateFormat = moment().toMomentFormatString(input.attr('data-dateformat'));
-
         input.daterangepicker({
+          parentEl: "#"+this.parentElement.id,
```

Comment:
Doesn't this id need escaping to be used in the selector?

Reply:
I'm not sure I understand your point. We get the id from the parentElement. If this id was not escaped before it'd break beforehand. E.g. there can't be any spaces in this id since it's already used as an id.
I think this'd cause double escaping.

Reply:
You're passing this as selector in jQuery here I think. The id may contain, e.g., `.`. If you don't escape this, the selector won't match the id but something else. There seems to be https://api.jquery.com/jQuery.escapeSelector/ for this purpose.

Reply:
Thanks a lot! 
It's clearer to me now :)
This same structure is also found [here in the xwiki-platform repository](https://github.com/xwiki/xwiki-platform/blob/6006c3c4ae08e592d12de3230a7cc5cb06dd1f1a/xwiki-platform-core/xwiki-platform-edit/xwiki-platform-edit-ui/src/main/resources/XWiki/InplaceEditing.xml#L407).
Fixed in https://github.com/xwiki/xwiki-platform/pull/2149/commits/3e6c82bd0a4ae2ce86c77400708688b72c955618